bitnami/influxdb/templates/influxdb/statefulset-high-availability.yaml

```diff
@@ -64,13 +64,25 @@ spec:
           image: {{ include "influxdb.volumePermissions.image" . }}
           imagePullPolicy: {{ .Values.volumePermissions.image.pullPolicy | quote }}
           command:
-            - sh
-            - -c
+            - /bin/bash
+            - -cx
             - |
+              {{- if .Values.persistence.enabled }}
```

Comment:
This is not necessary. You're already checking this is `true` on line 61.

Reply:
Fixed in latest commit